losses/dice.py

```diff
@@ -1,35 +1,27 @@
-import torch
 import torch.nn as nn
+from catalyst.utils import metrics
```

Comment:
btw, could we transfer metrics from utils to dl? looks like they finally become DL specific ones
and utils are too complex already